src/components/DelistCard/DelistCard.tsx

```diff
@@ -137,9 +137,11 @@ const DelistCard: React.FC<DelistCardProps> = ({ isOpen, close, nft, auction })
                     delist()
                     setDelistSuccess(true)
                   }}
+                  disabledBackground="#c2c2c2"
```

Comment:
should be move styling to styled component

Reply:
![image](https://user-images.githubusercontent.com/23531140/131837315-bd82fd3d-d41d-4e9f-8017-857cf78894cb.png)
this is dynmic property of delistButtton